src/firebase.js

```diff
@@ -0,0 +1,15 @@
+import firebase from 'firebase/compat/app';
+import 'firebase/compat/auth';
+
+const firebaseConfig = {
+    apiKey: "AIzaSyBhzSlAhXv1CnCnyMVzTNQCIvyg6Jrr8aM",
+  authDomain: "leeway-staging-f2db3.firebaseapp.com",
+  projectId: "leeway-staging-f2db3",
+  storageBucket: "leeway-staging-f2db3.appspot.com",
+  messagingSenderId: "336011377473",
+  appId: "1:336011377473:web:90af19959b9b3e5b4c7fc0"
```

Comment:
Serious tip: Never store secret keys in code base. We should follow a environment file pattern so that its not exposed on repos.
I will share you some materials on best practices for this.